src/stream/src/executor/flow_control.rs

```diff
@@ -50,7 +50,10 @@ impl FlowControlExecutor {
             let clock = MonotonicClock;
             RateLimiter::direct_with_clock(quota, &clock)
         };
+        #[cfg(not(madsim))]
         let rate_limiter = self.rate_limit.map(get_rate_limiter);
+        #[cfg(madsim)]
+        let mut rate_limit_quota = self.rate_limit.unwrap_or(0);
```

Comment:
`0` - is this expected?

Reply:
It's just an arbitrary number. Only when `rate_limit` is present, will `rate_limit_quota` be used.

Reply:
Not used anymore.